components/visgl/content/frameworks.json

```diff
@@ -1,86 +1,122 @@
           "image": "/visgl/images/frameworks/deck.png",
-          "description": "Providing high-performance, GPU powered visualization layers for  large scale geospatial data. deck.gl is the corner stone of the vis.gl framework suite. A selection of submodules provide layers for various geospatial and 3D  use cases.\n"
-        },
-        {
-          "name": "pydeck.gl",
-          "url": "https://pydeck.gl",
-          "image": "/visgl/images/frameworks/pydeck.png",
-          "description": "Python bindings for deck.gl that enable powerful custom geospatial visualizations to be programmed directly in Jupyter Notebooks.\n"
+          "description": "High-performance, GPU powered visualization layers for large scale geospatial data. Submodules provide layers for various geospatial and 3D  use cases.\n"
         },
         {
           "name": "loaders.gl",
           "url": "https://loaders.gl",
           "image": "/visgl/images/frameworks/loaders.png",
-          "description": "A wide range of highly optimized, framework-independent loaders for geospatial,  tabular and 3D file formats. Supports worker based binary data loading. Supported outputs includes geospatial layers, point clouds, 3D geometries,  images, textures and tabular data.\n"
+          "description": "Optimized, framework-independent loaders for geospatial, tabular and 3D file formats. Supports worker-based, binary data loading.\n"
         },
         {
-          "name": "luma.gl",
-          "url": "https://luma.gl",
-          "image": "/visgl/images/frameworks/luma.png",
-          "description": "A high-performance WebGL library for high-performance visualization and GPU computation.\n"
-        },
-        {
-          "name": "hubble.gl",
-          "url": "https://hubble.gl",
-          "image": "/visgl/images/frameworks/hubble.png",
-          "description": "A powerful animation and video capture solution for deck.gl and kepler.gl visualizations.\n"
+          "name": "pydeck.gl",
+          "url": "https://pydeck.gl",
+          "image": "/visgl/images/frameworks/pydeck.png",
+          "description": "Python bindings for deck.gl that enable custom geospatial visualizations to be programmed directly in Jupyter Notebooks.\n"
         },
         {
           "name": "react-map-gl",
           "url": "https://visgl.github.io/react-map-gl",
           "image": "/visgl/images/frameworks/react-map-gl.png",
-          "description": "A React wrapper for Mapbox GL JS that integrates with deck.gl.\n"
+          "description": "A React wrapper for Maplibra and Mapbox GL JS that integrates with deck.gl.\n"
         },
         {
           "name": "react-google-maps",
           "url": "https://visgl.github.io/react-google-maps/",
           "image": "/visgl/images/frameworks/react-google-maps.jpeg",
           "description": "A React wrapper for the Google Maps JavaScript API that integrates with deck.gl.\n"
         },
+        {
+          "name": "hubble.gl",
```

Comment:
I envision video and animation as a common capability, but agree with its grouping here for now snow it primarily serves geospatial use cases